packages/roosterjs-editor-plugins/lib/plugins/Announce/AnnounceHandlerImpl.ts

```diff
@@ -0,0 +1,69 @@
+import type { AnnounceData, DefaultAnnounceStrings } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ */
+export default class AnnounceHandler {
+    private ariaLiveElement: HTMLDivElement | undefined;
+
+    constructor(
+        private readonly document: Document,
+        private stringsMap?: Map<DefaultAnnounceStrings, string>
```

Comment:
Do we want to support switching language when plugin is still alive? Maybe not a hard requirement.

Reply:
How do you think it would be best? 

I can think of: 
1. Adding a public function to the plugin to set the string map
2. Something similar to setZoomScale.

Reply:
Maybe no need to support it now. But if we want, we can do something like

```ts
constructor(..., stringMap: Map<...> | (key) => string)
```

So that we can provide a callback function to return the string.

Reply:
Sounds good, added the callback